.travis.yml

```diff
@@ -1,12 +1,27 @@
 language: node_js
 node_js:
   - "14"
+
 notifications:
   email: false
+
 install:
   - npm install
-script:
-  - npm test
-  - npm run audit
-  - npm run doc
-  - npm run lint
+  - npx lerna bootstrap
```

Comment:
There is no need to call `npx lerna bootstrap`, since it is part of postinstall script.

Reply:
Removed.